### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by kazaneza**
### In this pull request

- [ ]  I build a simple HTML list of To Do tasks.

- [ ]  installed the webpack

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        const taskDescription = document.createElement('span');
+        taskDescription.innerHTML = task.description;
+        const taskActions = document.createElement('div');
+        taskActions.innerHTML = '...';
+        listItem.appendChild(checkbox);
+        listItem.appendChild(taskDescription);
+        listItem.appendChild(taskActions);
+        taskList.appendChild(listItem);
+      });
+  };
```
Comment (DammyShittu) [inline src/index.js:26]: - [ ] Kindly align the text of the tasks to the left as we have on the design.


  Your design | Design
  :---:|:---:
  yue | kdkd

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @kazaneza,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (GedeonTS) [inline src/index.js:26]: -[OPTIONAL] Good job making the changes requested by the previous reviewer [here](https://github.com/kazaneza/to-do-list/pull/1/files#r1089485743), However, he forgot to mention that you had to create a clone of the provided todo list image, Kindly use similar colors and reduce the paddings to fix that :+1:

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Webpack</title>
+    <!-- <link rel="stylesheet" href="style.css"> -->
```
Comment (GedeonTS) [inline src/index.html:7]: - [ OPTIONAL] Kindly make sure to delete commented out lines of code to make your project clean :recycle:

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @kazaneza ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good use of GitHub flow :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**